ex2_hal/northern_spirit/equipment_handler/include/northern_voices_games.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2021  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+/**
+ * @file northern_voices_games.h
+ * @author Thomas Ganley
+ * @date 2022-06-12
+ */
+
+#ifndef EX2_HAL_NORTHERN_SPIRIT_EQUIPMENT_HANDLER_INCLUDE_NORTHERN_VOICES_GAMES_H_
+#define EX2_HAL_NORTHERN_SPIRIT_EQUIPMENT_HANDLER_INCLUDE_NORTHERN_VOICES_GAMES_H_
+
+#include "northern_spirit_handler.h"
+#include <redposix.h>
```

Comment:
Headers should only have the bare minimum includes, I don't see any reliance edge stuff in a few of these, headers, should remove them